content/blog/2022-06-28-serving-models-with-mlem.md

```diff
@@ -0,0 +1,277 @@
+---
+title: Serving Models with MLEM
+date: 2022-06-28
+description: >
+  Once you have a model that's ready for production, getting it out can be a
+  complicated step. In this tutorial, we're going to use MLEM to deploy a model
+  to an API.
+descriptionLong: >
+  Getting models out to our end users is usually a difficult task. You have to
+  make sure it's in the correct format for the environment and for the way it
+  will be consumed. We're going to walk through how you can use MLEM to save
+  your model and serve it through an API.
```

Comment:
```suggestion
  Getting a machine learning model out to our end users can be a difficult task. You have to
  make sure it's in the correct format for the environment and for the way it
  will be consumed. We're going to walk through how you can use MLEM to save
  your model and serve it with a web API.
```

> can be a difficult task

Or if we could say "doesn't have to be a difficult task", that could be a better hook.